bundle/edu.gemini.itc.shared/src/main/scala/edu/gemini/itc/shared/ConfigExtractor.scala

```diff
@@ -123,7 +120,7 @@ object ConfigExtractor {
       slitWidth   <- extract[SlitWidth]         (c, SlitWidthKey)
       grating     <- extractDisperser
       camera      <- extractCamera
```

Comment:
Is there some reason we're not doing this change in extraction for other things like `camera` as well?
`camera <- extract[Camera](c, CameraKey)` ?